<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDetailsWidget</class>
 <widget class="QTabWidget" name="TableDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>446</height>
   </rect>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::West</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabStructure">
   <attribute name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/layer-database</normaloff>:/16/layer-database</iconset>
   </attribute>
   <attribute name="title">
    <string>Structure</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblTableInfo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableStructure">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnUndo">
            <property name="text">
             <string>&amp;Undo</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/16/edit-undo</normaloff>:/16/edit-undo</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnApply">
            <property name="text">
             <string>&amp;Apply</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/16/dialog-ok-apply</normaloff>:/16/dialog-ok-apply</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableIndexes">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabData">
   <attribute name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/database-table</normaloff>:/16/database-table</iconset>
   </attribute>
   <attribute name="title">
    <string>Data</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblQueryDesc">
        <property name="text">
         <string notr="true">SELECT * FROM %1 WHERE</string>
        </property>
        <property name="buddy">
         <cstring>txtWhere</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtWhere"/>
      </item>
      <item>
       <widget class="QLabel" name="lblQueryLimit">
        <property name="text">
         <string>LIMIT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="txtLimit">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefreshData">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/16/database-refresh</normaloff>:/16/database-refresh</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEditQuery">
        <property name="text">
         <string>Edit Query</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/16/database-edit</normaloff>:/16/database-edit</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="EditableTableView" name="tableData">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditableTableView</class>
   <extends>QTableView</extends>
   <header>.EditableTableView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableStructure</tabstop>
  <tabstop>txtWhere</tabstop>
  <tabstop>btnRefreshData</tabstop>
  <tabstop>btnEditQuery</tabstop>
  <tabstop>tableData</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
